cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3778,7 +3818,13 @@ class TranslatedLambdaExpr extends TranslatedNonConstantExpr, InitializationCont
     )
     or
     tag = LoadTag() and
-    result = this.getParent().getChildSuccessor(this, kind)
+    (
+      result = this.getParent().getChildSuccessor(this, kind)
+      or
+      // All load instruction could throw an SEH exception
+      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge e), true) and
+      kind.(ExceptionEdge).isSEH()
```

Comment:
This load is an implementation detail to model lambdas. I don' think these can ever throw a SEH exception.

Reply:
Done